dipy/direction/closest_peak_direction_getter.pyx

```diff
@@ -1,16 +1,24 @@
 import numpy as np
 cimport numpy as cnp
+cimport cython
```

Comment:
I do not think this import is needed. This is not used

Reply:
yes